### DISCUSSION TITLE
Mixed Language Usage?

### DISCUSSION THREAD

**Discussion by wubinbai** [coqui-ai/TTS#1924] [General]
Mix Language Usage? Is it possible to use tts for a text with mixed language? e.g. Chinese and English simultaneously

**Comment by erogol** [comment]
yes you can use YourTTS

**Comment by pkuhsczy** [comment]
How to accomplish this task? Do you need to write the underlying code yourself?

**Comment by erogol** [comment]
dyou can give texts separately to the model and set the language accordingly. If you mean to change the language in the sentence, it is not supported yet. You need to write your own code for your target model.

**Reply by HT5757** [reply]
You know if is supported now? Is there a any TTS which you can recommend?
